packages/manager/modules/pci/src/projects/project/storages/containers/container/addUser/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,18 @@
+{
+  "pci_projects_project_storages_containers_container_addUser_archive_title": "Ajouter un utilisateur à mon conteneur",
+  "pci_projects_project_storages_containers_container_addUser_object_title": "Ajouter un utilisateur à mon conteneur",
+
+  "pci_projects_project_storages_containers_container_addUser_description_step_0": "Sélectionnez l’utilisateur à ajouter à votre conteneur. S’il n’apparaît pas dans cette liste, veuillez le créer dans la section dédiée.",
+  "pci_projects_project_storages_containers_container_addUser_description_step_1": "Définissez le rôle de votre utilisateur sur le conteneur {{container}}.",
+
+  "pci_projects_project_storages_containers_container_addUser_submit_label": "Confirmer",
+  "pci_projects_project_storages_containers_container_addUser_cancel_label": "Annuler",
+
+  "pci_projects_project_storages_containers_container_addUser_right_admin": "Administrator",
+  "pci_projects_project_storages_containers_container_addUser_right_readWrite": "Read and write",
```

Comment:
```suggestion
  "pci_projects_project_storages_containers_container_addUser_right_readWrite": "Lecture & Écriture",
```

Reply:
The PU wants these roles in english and not translated.

Reply:
If some content are not supposed to be translated, then it shouldn't supposed to be store into a translation file.